recipes/flask-mail/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "flask-mail" %}
+{%set camelName = "Flask-Mail" %}
```

Comment:
We should really sit down with `conda skeleton pypi` and get that to behave better. Ideally we should just be able to churn out recipes with it and use them without (or with little) modification.